airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -260,6 +265,24 @@ def _wait_for_task_ended(self) -> None:
         waiter.config.max_attempts = sys.maxsize  # timeout is managed by airflow
         waiter.wait(cluster=self.cluster, tasks=[self.arn])
 
+    def _cloudwatch_log_events(self):
+        if not self.awslogs_group:
+            return
+
+        if not self.awslogs_stream_prefix:
+            return
+
+        task_id = self.arn.split("/")[-1]
+        stream_name = "{}/{}".format(self.awslogs_stream_prefix, task_id)
```

Comment:
We have started to enforce using f-string, and this `.format()` usage should fail the CI.

Reply:
Fixed `stream_name`